modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -5,72 +5,144 @@
+
+You can also edit an existing `ContainerRuntimeConfig` CRs to edit existing settings or add new settings instead of creating a new CR. It is recommended to create a new CR to modify a different machine config pool, rather than add additional labels to an existing CR.
+
+If you want to delete the machine configs, you should delete them in reverse order to avoid exceeding the limit. For example, you should delete the `containerruntime-3` machine config before deleting the `containerruntime-2` machine config. 
+
+[NOTE]
+====
+If you have a machine config with a `containerruntime-9` suffix, the next machine config is appended with `containerruntime-10` and will fail as it exceeds the limit, even if there are fewer than 10 `containerruntime` machine configs.
+====
+
+.Example showing multiple `ContainerRuntimeConfig` CRs
+[source,terminal]
+----
+$ oc get ctrcfg
+
+NAME         AGE
+ctr-pid      24m
+ctr-overlay  15m
+ctr-level    5m45s
+----
+
+.Example showing multiple `containerruntime` machine configs
+[source,terminal]
+----
+$ oc get mc | grep container
 
+...
+01-master-container-runtime                        b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             57m
+...
+01-worker-container-runtime                        b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             57m
+...
+99-worker-generated-containerruntime               b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             26m
+99-worker-generated-containerruntime-1             b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             17m
+99-worker-generated-containerruntime-2             b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             7m26s
+...
+----
+
+The following example raises the `pids_limit` to 2048, sets the `log_level` to `debug`, sets the overlay size to 8 GB, and sets the `log_size_max` to unlimited:
+
+.Example `ContainerRuntimeConfig` CR
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: overlay-size
+spec:
+ machineConfigPoolSelector:
+   matchLabels:
+     pools.operator.machineconfiguration.openshift.io/worker: ''
+ containerRuntimeConfig:
+   pidsLimit: 2048 <2>
+   logLevel: debug <3>
+   overlaySize: 8G <4>
+   logSizeMax: "-1" <5>
+----
+<1> Specifies the machine config pool label.
+<2> Optional: Specifies the level of verbosity for log messages.
+<3> Optional: Specifies the maximum size allowed for the container log file. If set to a positive number, it must be at least `8192`.
+<4> Optional: Specifies the maximum size of a container image. 
+<5> Optional: Specifies the maximum number of processes allowed in a container.
+	
 .Procedure
 
-. To raise the `pidsLimit` to 2048, set the `logLevel` to `debug`, and set the `overlaySize` to 8 GB, create a CR file (for example, `overlay-size.yaml`) that contains that setting:
+To change CRI-O settings using the `ContainerRuntimeConfig` CR.
```

Comment:
Follow this procedure to change (otherwise this sentence fragment is just dangling there)